doc/3/controllers/collection/truncate/index.md

```diff
@@ -0,0 +1,33 @@
+---
+code: true
+type: page
+title: truncate
+description: Remove all documents from collection
+---
+
+# truncate
+
+Removes all documents from a collection, while keeping the associated mapping.
+
+<br/>
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> truncate(
+      final String index,
+      final String collection) throws NotConnectedException, InternalException
+```
+
+<br/>
+
+| Arguments    | Type              | Description     |
+| ------------ | ----------------- | --------------- |
+| `index`      | <pre>String</pre> | Index name      |
+| `collection` | <pre>String</pre> | Collection name |
+
+## Returns
+
+A `ConcurrentHashMap` which has an `acknowledged` Boolean.
```

Comment:
IMHO, you can return nothing

Reply:
The API returns it
https://docs.kuzzle.io/core/2/api/controllers/collection/truncate/

Reply:
Yes but it's useless IMHO

Reply:
The API returns an acknowledgement. This is a way of saying: "the query has been accepted and it ran fine, but there is no result to return".

So, in a SDK, this is a same than a function returning normally. Bothering with creating a ConcurrentHashMap just to return an acknowledgement has no point.

Reply:
And to me, this warrants a "requested change", so I'm going to block this PR over this issue.